engines/engines.php

```diff
@@ -227,7 +227,7 @@ function engine_setup_meta($engine, $meta)
 {
     global $config;
 
-    if (is_array($meta['capabilities'])) {
+    if (array_key_exists( 'capabilities', $meta )) {
```

Comment:
Same here, should be:
`if (array_key_exists('capabilities', $meta) && is_array($meta['capabilities']))`
to keep current functionality & ...

Reply:
code amended